shared/src/main/scala/com/cibo/evilplot/demo/DemoPlots.scala

```diff
@@ -200,39 +149,39 @@ object DemoPlots {
       .render(plotAreaSize)
   }
 
-  lazy val facetedPlot: Drawable = {
-    val years = 2007 to 2013
-    val datas: Seq[Seq[Point]] =
-      years.map(_ => Seq.fill(Random.nextInt(20))(Point(Random.nextDouble(), Random.nextDouble())))
-    val plot1 = Overlay(ScatterPlot.series(datas(0), "2010", HTMLNamedColors.red))
-    val plot2 = Overlay(
-      ScatterPlot.series(datas(0), "2010", HTMLNamedColors.red),
-      ScatterPlot.series(datas(1), "2011", HTMLNamedColors.blue)
-    )
-    val plot3 = Overlay(
-      ScatterPlot.series(datas(0), "2010", HTMLNamedColors.red),
-      ScatterPlot.series(datas(1), "2011", HTMLNamedColors.blue),
-      ScatterPlot.series(datas(2), "2012", HTMLNamedColors.green),
-      ScatterPlot.series(datas(3), "2013", HTMLNamedColors.teal)
-    )
-    val plot4 = Overlay(
-      ScatterPlot.series(datas(0), "2010", HTMLNamedColors.red)
-    )
+  lazy val functionPlot: Drawable = {
+    val Seq(one, two, three) = theme.colors.stream.take(3)
+    Overlay(
+      FunctionPlot.series(x => x * x, "y = x\u00B2", one, xbounds = Some(Bounds(-1, 1))),
+      FunctionPlot.series(x => math.pow(x, 3), "y = x\u00B3", two, xbounds = Some(Bounds(-1, 1))),
+      FunctionPlot.series(x => math.pow(x, 4), "y = x\u2074", three, xbounds = Some(Bounds(-1, 1)))
+    ).title("A bunch of polynomials.")
+      .overlayLegend()
+      .standard()
+      .render(plotAreaSize)
+  }
 
-    Facets(Seq(Seq(plot1, plot2), Seq(plot3, plot4)))
+  lazy val boxPlot: Drawable = {
+    val data = Seq.fill(10)(Seq.fill(Random.nextInt(30))(Random.nextDouble()))
+    BoxPlot(data)
+      .standard(xLabels = (1 to 10).map(_.toString))
+      .render(plotAreaSize)
+  }
+
+  lazy val scatterPlot: Drawable = {
+    val points = Seq.fill(150)(Point(Random.nextDouble(), Random.nextDouble())) :+ Point(0.0, 0.0)
+    val years = Seq.fill(150)(Random.nextDouble()) :+ 1.0
+    ScatterPlot(points, pointRenderer = Some(PointRenderer.depthColor(years, None, None)))
       .frame()
+      .xGrid()
```

Comment:
`.xGrid().yGrid().xAxis().yAxis().frame()` should be equivalent to `.standard()`